clients/java/src/main/java/io/camunda/zeebe/client/impl/search/SearchResponseMapper.java

```diff
@@ -47,10 +50,32 @@ public static SearchQueryResponse<ProcessInstance> toProcessInstanceSearchRespon
                 (i) ->
                     i.stream()
                         .map(ProcessInstanceImpl::new)
-                        .map((p) -> (ProcessInstance) p)
+                        .map(ProcessInstance.class::cast)
                         .collect(Collectors.toList()))
             .orElse(Collections.emptyList());
 
     return new SearchQueryResponseImpl<>(instances, page);
   }
+
+  public static SearchQueryResponse<UserTaskItem> toUserTaskSearchResponse(
+      final UserTaskSearchQueryResponse response) {
+    final SearchQueryPageResponse pageResponse = response.getPage();
+    final SearchResponsePage page =
+        new SearchResponsePageImpl(
+            pageResponse.getTotalItems(),
+            pageResponse.getFirstSortValues(),
+            pageResponse.getLastSortValues());
```

Comment:
:x: Please extract this in its own method so that it can be reused in `toProcessInstanceSearchResponse()` and here.